estuary_updater/handlers/base.py

```diff
@@ -89,42 +89,52 @@ def is_container_build(self, build_info):
         else:
             return False
 
-    def get_or_create_build(self, identifier, original_nvr=None, force_container_label=False):
+    def get_or_create_build(self, identifier, build_info=None, original_nvr=None,
+                            force_container_label=False):
         """
         Get a Koji build from Neo4j, or create it if it does not exist in Neo4j.
 
         :param str/int identifier: an NVR (str) or build ID (int)
+        :param dict build_info: all info needed to create a build, else get from a Koji session
```

Comment:
This is actually a `kwarg` and not a `param`.

I'd also replace `else get from a Koji session` with `else get it from Koji`